app/assets/stylesheets/styles/components/guides/_edit_guides.css.scss

```diff
@@ -173,7 +173,7 @@
 
   */
   @media only screen and (max-width: 40.063em){
-    .requirements .actual-requirements{
+    .basic-needs .actual-basic-needs{
```

Comment:
Selector should have depth of applicability no greater than 2, but was 3<br>Opening curly brace `{` should be preceded by one space